design-proposals/klip-15-new-api-and-client.md

```diff
@@ -133,14 +132,14 @@ Push queries can be explicitly terminated by the client by making a request to t
 
 The request method will be a POST.
 
-Requests will be made to a specific URL, e.g. "/query-terminate" (this can be configurable)
+Requests will be made to a specific URL, e.g. "/query-close" (this can be configurable)
```

Comment:
It's `close-query` and not `query-close`, right?

Also, what does "(this can be configurable)" mean?